client.js

```diff
@@ -43,16 +42,20 @@ async function sqlExample(hzClient) {
             'keyFormat'='varchar',
             'valueFormat'='varchar'
         );`;
-    await hzClient.getSql().execute(mappingQuery);
+    await client.getSql().execute(mappingQuery);
     console.log("The mapping has been created successfully.");
     console.log("--------------------");
+}
 
+async function clearCapitals(client){
     console.log("Deleting data via SQL...");
     const deleteQuery = "DELETE FROM cities";
```

Comment:
You need to use `capitals` instead of cities in SQL in all methods. See the [java sample ](https://github.com/hazelcast/hazelcast-cloud-java-sample-client/blob/5.1/src/main/java/com/hazelcast/cloud/Client.java#L107)